src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/AdjournmentCalculateDateHelper.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
```

Comment:
This could also be in a helper package

Reply:
I have created a new package "adjournment" in mapping, then put this helper class and the two enums in there.